docs/gatsby-config.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
           // For any node of type MarkdownRemark, list how to resolve the fields` values
           MarkdownRemark: {
             title: node => node.frontmatter.title,
-            slug: node => `#octokit-${node.fields.idName}`,
+            slug: node => `#${node.fields.idName}`,
```

Comment:
I think in that case we can just do 

```suggestion
            slug: node => node.fields.idName,
```

Reply:
It looks like in the `SearchResults` component it drops in the `slug` value directly into the `href` attribute, so I think we still need the hash prefix in order to link to the anchor. The other slug definitions for `OctokitApiGroup` and `OctokitApiMethod` do this too.

Reply:
I'm sorry I don't follow, I just wonder if we need to wrap `node.fields.idName` into a template string? Or do you refer to something else? I'm lacking a bit context right now, sorry

Reply:
Oh sorry I was talking about how we need the `#` at the start of the template string. `node.fields.idName` doesn't include that so that needs to be prepended to idName to end up with `#previews` or `#usage`.

I'm okay with doing `'#' + node.fields.idName` too.

Reply:
🤦‍♂ sorry I totally missed the `#`, I better get a coffee 😅